app/bower_components/angular-cryptography/angular-cryptography.d.ts

```
@@ -0,0 +1,17 @@
+// Type definitions for AngularCryptography (mdo-angular-cryptography module)
+// Project: https://github.com/middleout/angular-cryptography
+// Definitions by: Andrei Gabreanu <andrei.gabreanu@middleout.com>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+declare module mdo.cryptography {
```

Biome (1.9.4)

[error] Use the namespace keyword instead of the outdated module keyword.

The module keyword is deprecated to avoid any confusion with the ECMAScript modules which are often called modules.
Safe fix: Use namespace instead.


(lint/suspicious/useNamespaceKeyword)